tracer/test/Datadog.Trace.Security.Unit.Tests/WafCompatibilityTests.cs

```diff
@@ -3,25 +3,21 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-using System.Collections.Specialized;
-using Datadog.Trace.AppSec;
-using Datadog.Trace.AppSec.Waf;
 using Datadog.Trace.AppSec.Waf.NativeBindings;
-using Datadog.Trace.Configuration;
 using FluentAssertions;
 using Xunit;
 
 namespace Datadog.Trace.Security.Unit.Tests
 {
-    [Collection("WafTests")]
     public class WafCompatibilityTests
     {
         [SkippableFact]
         public void ShouldNotInitialize()
         {
-            var initializationResult = WafLibraryInvoker.Initialize("1.4.0");
-            initializationResult.Success.Should().BeFalse();
-            initializationResult.ExportErrorHappened.Should().BeTrue();
+            var libraryInitializationResult = WafLibraryInvoker.Initialize("1.4.0");
```

Comment:
The static `WafLibraryInvoker.Initialize()` method won't cause concurrency issues with other tests?

Reply:
I'm not sure why.. cause they should be loaded separately and within the singleton now 🤔